mixed-content/generic/expect.py

```diff
@@ -95,7 +95,7 @@ def main(request, response):
 
     response.add_required_headers = False
     response.writer.write_status(200)
-    response.writer.write_header("content-type", content_type)
-    response.writer.write_header("cache-control", "no-cache; must-revalidate")
+    response.writer.write_header("Content-Type", content_type)
+    #response.writer.write_header("cache-control", "no-cache; must-revalidate")
```

Comment:
Did you mean to comment this out? And in the line above, does the capitalization of Content-Type matter? O.o

Reply:
Nope, didn't mean to commit this at all. I was poking at things to see if they would make Safari happy; this wasn't it.